docs/how-to-guides/debug-builds.rst

```diff
@@ -0,0 +1,49 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+.. _how-to-guides-debug-builds:
+
+******************
+Using debug builds
+******************
+
+For development work on native code in your Python package, you may want to use
+a debug build. To do so, we need to use the ``buildtype`` argument, which is
```

Comment:
Isn't it?
```suggestion
a debug build. To do so, we need to use the ``buildtype=debug`` argument, which is
```

Reply:
I was calling `buildtype` the argument, and I'd consider `debug` its value. The full invocation is given on the `pip install` line further down.

I don't have a strong preference here - the way I have it now seems more concise to me, but it could also be `buildtype=debug` or `-Dbuildtype=debug`.

Reply:
I would write "buildtype option" (preferred) or "-Dbuildtype=debug argument"

Reply:
Updated to "`buildtype` option"

Reply:
Sorry, perhaps I was unclear; you have "the ``buildtype`` option, which is equivalent to ``-Ddebug=true -Doptimization=0``", which is not directly exactly true. It is `buildtype=debug` which is equivalent to those two settings; `buildtype` by itself is equivalent to many combinations of those options.

Reply:
Thanks. Indeed, the sentence as it reads now is not precise. I'll fix this.

Reply:
Good point @QuLogic, thanks. The fix is included as part of gh-476.